cloudant-client/src/test/java/com/cloudant/tests/HttpTest.java

```diff
@@ -562,6 +563,9 @@ private void testInputStreamRetry(HttpConnection request, byte[] expectedContent
                 try {
                     if (444 == context.connection.getConnection().getResponseCode()) {
                         context.replayRequest = true;
+                        // Close the error stream
+                        InputStream errors = context.connection.getConnection().getErrorStream();
+                        if (errors != null) IOUtils.closeQuietly(errors);
```

Comment:
This could be troublesome if we want to update our commons-io dependency, which is a bit out of date.

https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/IOUtils.html#closeQuietly(java.io.InputStream)

```
Deprecated. As of 2.6 removed without replacement. Please use the try-with-resources statement or handle suppressed exceptions manually
```

It looks like this method is also null-guarded, although that feature isn't documented.

Reply:
Fair point. We use this in quite a few other places as well. I'll common them up.

Reply:
Done in 0687f06cc3dee69903fac1e48661151656056a32. Note just for the cloudant-http module we still have some usages of `closeQuietly` in tests and cloudant-client.